nested-labvm/atd-docker/docker-compose.yml

```diff
@@ -194,6 +196,14 @@ services:
       - "/home/arista/arista-dir/apps/webui:/config:rw"
     networks:
       - atd_nginx
+  grader:
+    container_name: atd-grader
+    image: us.gcr.io/atd-testdrivetraining-dev/atddocker_selfgrading:latest
```

Comment:
Need to change this to a deployable tag

Reply:
We do need a method to handle feature tags like this, I have one for live chat.